electron/app/commands.ts

```diff
@@ -136,11 +135,6 @@ export const runCommand = (options: CommandOptions, event: Electron.IpcMainEvent
 
   const result: CommandResult = {exitCode: null, signal: null};
 
-  fixPath();
```

Comment:
removed it here since it is called when creating the window

Reply:
I was thinking of adding it to the `electron/main.ts` before everything else loads just to make sure all/any future features that we'll add and might depend on the PATH variable will work well

Reply:
thanks @devcatalin I will move it there - how can I test that it works?